doc/source/CHANGELOG.rst

```diff
@@ -18,6 +18,8 @@ Bugfixes
   working when the number didn't fit in 'q'.
 - Fix mapped_tuple pack_into. If long headers were used, it would
   unintentionally expand the given buffer.
+- Fix requirements to exclude numpy 1.17.0 and above,
```

Comment:
Shouldn't this go on an unreleased version?

Reply:
I just added it to the last one assuming it was the unreleased. My bad.